ProcessLib/StaggeredCouplingTerm.cpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   StaggeredCouplingTerm.cpp
+ *
+ * Created on November 7, 2016, 12:14 PM
+ */
+
+#include "StaggeredCouplingTerm.h"
+#include "Process.h"
+
+namespace ProcessLib
+{
+const StaggeredCouplingTerm createVoidStaggeredCouplingTerm()
```

Comment:
It means an empty instance of StaggeredCouplingTerm. There is a documentation for this function.